src/modules/auth/index.js

```diff
@@ -42,23 +44,28 @@ export default class AuthModule extends Module {
 		
 		// Do we have an oauth token (we are authorized to use the app)
 		if (this.OAuthToken != null) {
-
 			// Update the user info
 			let result = await this._updateUserInfo();
 			if (!result) {
 				// User info not found - go to login
 				sessionStorage.removeItem('SeaCatOAuth2Token');
 				let force_login_prompt = true;
 
-				this.Api.login(this.RedirectURL, force_login_prompt);
+				this.SeaCatAuthApi.login(this.RedirectURL, force_login_prompt);
+				return;
+			}
+			// Authorization of the user based on rbac
+			let userAuthorized = await this._isUserAuthorized();
```

Comment:
I guess this has to be configurable - we have applications with authentication but without autorization.